medical_compliance/medical_compliance/api/tasks.py

```diff
@@ -74,6 +74,6 @@ def fetch_heart_rate_measurement():
         )
         heart_rate_measurement.save()
 
-    analyze_heart_rates.delay()
+    analyze_heart_rates.delay(last_meas)
```

Comment:
oh, sorry about just realized that it's an actual var you're using and not a typo